index.js

```diff
@@ -80,37 +81,66 @@ async function annotatePackage(packageName, filePath, lineNumber) {
     }
 }
 
-async function run() {
-    const filePath = 'requirements.txt';
-    const ecosystem = core.getInput('package-ecosystem', { required: true });
+async function processLines(filePath, lines) {
+    for (let index = 1; index <= lines.length; index++) {
+        const line = lines[index - 1]; // Keep lines including empty ones
 
-    if (ecosystem !== 'pip') {
-        core.setFailed(`Unsupported package ecosystem: ${ecosystem}`);
-        return;
+        if (!line.trim()) {
+            // Skip processing empty lines, but continue incrementing line count
+            continue;
+        }
+
+        const packageName = processPackageLine(line);
+        if (packageName) {
+            if (!isValidPackageName(packageName)) {
+                core.error(`Invalid package name: ${packageName}`, {
+                    file: filePath,
+                    startLine: index,
+                    endLine: index
+                });
+            } else {
+                await annotatePackage(packageName, filePath, index);
```

Comment:
This looks like it is use the pypi URL still

```
https://openteams-score.vercel.app/api/package/pypi/${
                                               ^^^^

Reply:
Yes, it does, I was unable to find another source: I have attempted requests to:
```
- package/conda
- package/anaconda
- package/conda-forge
```
And all returned 404. Is there an endpoint I am missing?

Reply:
sorry, this was not clear

it is `/package/conda-forge/{package-name}`

* https://openteams-score.vercel.app/api/package/pypi/flask
* https://openteams-score.vercel.app/api/package/conda/conda-forge/flask

Reply:
created https://github.com/openteamsinc/Score-Site/issues/24